bean-validation/3.0/_index.md

```diff
@@ -1,25 +1,24 @@
 ---
-title: "Jakarta Bean Validation 3.0 (Milestone)"
-date: 2020-06-09
+title: "Jakarta Bean Validation 3.0"
+date: 2020-07-07
```

Comment:
This is probably a discussion point for the Spec Committee.  Do we have a preference on what date these Final Specifications go live?  This `date:` field identifies when to allow this information be live on the Specifications page. Don't we want the ballots to be complete before they go live?  The earliest the ballots would be complete is Aug 03.  Or, is there a reason to hold these out for the final GA of the Platform?  I don't think so, but I'm throwing it out there for discussion.

Reply:
Upon further review, this should not be a concern.  These PRs should not be merged before the ballots complete, so there should not be an issue with pre-publishing data before the ballots are done.